modules/twitch_streaming

```diff
@@ -0,0 +1,58 @@
+
+"""
+Checks if a streamer is online. [Gross Gore or grossie_gore as example placeholder]
+
+Checks if a streamer is online using the requests library to check to see if the dictionary key 'stream' in the json response is set to None or not.
+
+Configuration parameters:
+    - cache_timeout : how often we refresh this module in seconds
+    - stream_name : name of streamer(twitch.tv/<streamer_name>)
+    
+Format of status string placeholders
+     - {stream_name}  :  name of the streamer (twitch.tv/<stream_name>)
+     - {status} : whether or not the streamer is online/offline
+     
+@author Alex Caswell horatioesf@virginmedia.com
+@license BSD
+"""
+
+# import your useful libs here
+from time import time
+import requests
+
+
+class Py3status:
+    # available configuration parameters
+    cache_timeout = 10
+    stream_name = "grossie_gore"
+    # can be customized in i3status.conf
+    format = "{stream_name} {status}"
+    format_offline = "{stream_name} {status}"
```

Comment:
I'm not sure that adding the `{status}` placeholder adds much here.  If there were more placeholders it'd make sense.

Others may have opinions.